public/assets/img/provider_logos/bitss/style.css

```diff
@@ -0,0 +1,135 @@
+a,
+.preprint-form-section header {
+  color: #1C4063; }
+
+.preprint-header {
+  background-image: none;
+  background-color: #e4f3ff;
+  color: black;
+  padding-top: 100px; }
+  .preprint-header .provider-brand {
+    height: 140px; }
+  .preprint-header a:not(.btn) {
+    color: black; }
+
+.navbar-image {
+  background-image: url("./square_color_transparent.png"); }
```

Comment:
As discussed, all image names and locations should be defined in the template rather than the CSS.